packages/grafana-ui/src/components/PageLayout/PageToolbar.tsx

```diff
@@ -63,24 +63,28 @@ export const PageToolbar: FC<Props> = React.memo(
             />
           </div>
         )}
-        {parent && parentHref && (
-          <>
-            <Link className={cx(styles.titleText, styles.parentLink, styles.titleLink)} href={parentHref}>
-              {parent} <span className={styles.parentIcon}></span>
-            </Link>
-            {titleHref && (
-              <span className={cx(styles.titleText, styles.titleDivider, styles.parentLink)} aria-hidden>
-                /
-              </span>
-            )}
-          </>
-        )}
-        {titleHref && (
-          <Link className={cx(styles.titleText, styles.titleLink)} href={titleHref}>
-            {title}
-          </Link>
-        )}
-        {!titleHref && <div className={styles.titleText}>{title}</div>}
+        <nav className={styles.navElement}>
+          {parent && parentHref && (
+            <>
+              <Link className={cx(styles.titleText, styles.parentLink, styles.titleLink)} href={parentHref}>
+                {parent} <span className={styles.parentIcon}></span>
+              </Link>
+              {titleHref && (
+                <span className={cx(styles.titleText, styles.titleDivider, styles.parentLink)} aria-hidden>
+                  /
+                </span>
+              )}
+            </>
+          )}
+          {titleHref && (
+            <h1 className={styles.titleText}>
```

Comment:
This causes issues with the link focus styles. Any suggestion on a solution would be appreciated!